mitosheet/src/components/endo/ColumnHeader.tsx

```diff
@@ -150,14 +154,21 @@ const ColumnHeader = (props: {
         />
     )
 
+    //If there is a dataRecon set, highlight the column headers that have been created or renamed
+    const isColumnCreated = reconIsColumnCreated(columnHeader, props.uiState, props.sheetData)
```

Comment:
This is an O(num_headers * num_created) operation (every header is checking against every created header). 

Note sure this a problem, but can you check on a huge dataset and see the performance impact?

Reply:
Its a bit hard to test on huge dataframes because of https://github.com/mito-ds/monorepo/issues/768, but its not an issue when we have 1000 rows and 40 columns.